### DISCUSSION TITLE
How do i call ShowDialog, without blocking every Window in the Program.

### DISCUSSION THREAD

**Discussion by BinarySignaling** [canton7/Stylet#367] [Q&A] [answered]
How do i use ShowDialog and not blocking every other window? At the moment if i call ShowDialog it blocks every other window that is open in the Program. If I close the Dialog all the Windows are working fine. 

Thank you in advance.

**Comment by canton7** [comment, marked as answer]
Use `ShowWindow` instead of `ShowDialog`. The point of a dialog is that it's modal, and blocks interaction with other windows.

**Reply by BinarySignaling** [reply]
But is there a way to block only one Window for the Dialog, or is the `ShowDialog(object viewModel, IViewAware ownerViewModel)` the owner Viewmodel not for this specific Window.

**Reply by canton7** [reply]
I don't believe that's possible in WPF (but I could be wrong).

Note that none of this is specific to Stylet -- `IWindowManager.ShowWindow` and `ShowDialog` just call into the [same methods on `Window`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.window.show?view=windowsdesktop-6.0), so if you can work out how do to it in raw WPF, just do the same in Stylet.
